### DISCUSSION TITLE
is it possible to replace the time by percentage on the wakatime card?

### DISCUSSION THREAD

**Discussion by Cableir** [anuraghazra/github-readme-stats#2948] [Q&A] [answered]
wakatime allows you to share charts with percentage option:
<img width="60%" src="https://wakatime.com/share/@23e3c62c-6593-4221-afb4-440323eb18e1/e47f8078-e08a-4816-b4b7-963980b4f388.svg">
but using github-readme-stats is it also possible to replace time by percentage?
![image](https://github.com/anuraghazra/github-readme-stats/assets/98467450/5ccf757e-05b7-4610-a7a5-bf349ddfda16)
(https://github-readme-stats.vercel.app/api/wakatime?username=gabrieldiaspereira&bg_color=2D3748&title_color=2F855A&icon_color=2F855A&text_color=ffffff&hide_title=true&langs_count=20&layout=compact)

**Comment by rickstaa** [comment, marked as answer]
@Cableir, we currently do not support the percentage format. Since it is a great addition to GRS, feel free to open a [pull](https://github.com/anuraghazra/github-readme-stats/compare) or [feature](https://github.com/anuraghazra/github-readme-stats/issues/new?assignees=&labels=enhancement&projects=&template=feature_request.yml) request.

**Comment by qwerty541** [comment]
I have opened feature request https://github.com/anuraghazra/github-readme-stats/issues/3016. so we can close this
